### DISCUSSION TITLE
How do we use "select a prompt"

### DISCUSSION THREAD

**Discussion by gerroon** [n4ze3m/page-assist#216] [Q&A] [answered]
How do I use the select a prompt feature?

![image](https://github.com/user-attachments/assets/155d0bf3-9b04-4582-a8cc-fb786ba4e8f4)

**Comment by n4ze3m** [comment, marked as answer]
It is a prompt dropdown that you can add from the 'Manage Prompt' section and use, for example, like this.

https://github.com/user-attachments/assets/5920daca-051b-45d0-bfa7-840df0e394b6

_Note: The selected system prompt won't work when chatting in website mode_

**Reply by gerroon** [reply]
Thanks you, I always use the sidebar so I never saw those options
